ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-SVC3B.cfg

```diff
@@ -160,8 +160,11 @@ interface Port-Channel7
    no shutdown
    switchport
    switchport trunk allowed vlan 110-112,120-121,130-131,140-141,150,160-161,210-211,250,310-311,350
+   switchport trunk native vlan tag
```

Comment:
This is odd

Reply:
This is indeed a valid command, as per our chat, let's create a PR to update the documentation in `eos_cli_config_gen`.